frontend/components/common/ProductCard.tsx

```diff
@@ -50,7 +50,7 @@ export const ProductCardImage = ({ src, alt }: ProductCardImageProps) => {
    return (
       <AspectRatio ratio={1} flexGrow={0} flexShrink={0} position="relative">
          <IfixitImage
-            sizes="30vw"
+            sizes="(max-width: 629px) 250px, (max-width: 767px) 400px, (max-width: 895px) 250px, (max-width: 1000px) 400px, 250px"
```

Comment:
This looks to me like it will use `250px` at over-`1000px` screen sizes. Is that correct?